buildscripts/azure/azure-linux-macos.yml

```
@@ -36,6 +36,16 @@ jobs:
       displayName: 'Flake8'
       condition: eq(variables['RUN_FLAKE8'], 'yes')
 
+    - script: |
+        export PATH=$HOME/miniconda3/bin:$PATH
+        conda install -y sphinx sphinx_rtd_theme pygments
+        echo "Building docs"
+        cd docs
+        SPHINXOPTS=-Wn clean html
```

```suggestion
        make SPHINXOPTS=-Wn clean html
```
think this is the problem with the "missing command"

---

yes, a `make` is missing